src/schemas/movies.py

```diff
@@ -1 +1,52 @@
+from typing import List
+from pydantic import BaseModel
+
+from routes import movies
+
 # Write your code here
+
+
+class MovieBase(BaseModel):
+    id: int
+    name: str
+    date: str
+    score: float
+    overview: str
+    status: str  # (Released | Post Production | In Production)
+    budget: float  # (>= 0)
+    revenue: float  # (>= 0)
+    country: str  # (ISO 3166-1 alpha-3 code)",
```

Comment:
According to the task description and models, the `country` should be a dictionary containing `id`, `code`, and `name`.

```suggestion
country: dict # (ISO 3166-1 alpha-3 code)
```